database/seed.js

```diff
@@ -4,7 +4,7 @@ const connection = require('../server/helper/restaurants.js');
 
 const restaurantSeed = () => {
```

Comment:
Note for this function and the rest of the functions in seed:
In general, I would have one of these functions for each table (not sure if you do already)
If a table has a foreign key referencing another table, I would make sure each item referenced by a primary key in the referenced table is being covered. For example, restaurants to popular dishes is a one-to-many relationship. Since popular dishes has a foreign key column that references a specific restaurant, it's important to make sure each restaurant is linked to some popular dishes in the restaurant table. One way to make sure of that would be to iterate over each restaurant, create 2 to 5 popular dishes for each restaurant, and make sure the restaurant_id foreign key references the parent restaurant. Restaurant 1 --> 2 to 5 popular dishes, each with restaurant_id field = 1. Then move onto restaurant 2 and do the same thing. 

That should be the approach for any relationship that's one to many. Others that come to mind are popular dishes(1) to photos(many). Each popular dish will have multiple photos, so iterate over the popular dishes and generate ~5 photos for each, with a foreign key reference to that popular dish's id. Let me know if this is confusing or if I explained it badly haha.